new-pipeline/dist.html

```diff
@@ -118,6 +118,25 @@ <h3><select id="selected-key4" class="multiselect" title="Selected key"></select
                                 </dl>
                             </div>
                         </div>
+                        <div id="stats-comparison" style="margin-top: 50px; margin-bottom: 20px;">
+                            <table>
+                              <thead class="measure-headers">
+                              <tr>
+                              <th></th>
+                              <th title="5th percentile duration">5 %</th>
```

Comment:
Is there a reason for leaving whitespace between the numbers and the '%'? I wouldn't want the unit to appear on a different line if the table layout algorithm has a bad day :S

Reply:
I'm a copy&paste HTML developer 😉 
In this case, it was copied from [this addon-perf table](https://github.com/mozilla/telemetry-dashboard/blob/7487b010f29252ce9f260f8250c16dd32e107b8d/addon-perf/index.html#L99)

I imagine it has a space because it's an abbreviation for **percentile**, not a unit.

Reply:
I always thought of % as a unit (like Hz, but instead of being "per second" it is "per hundred"). I will have to revise my understanding.

Reply:
I didn't mean in the general case, it's 100% a unit if considered in the common scenario. :) But then it reads as just percent.
Just in this context it looks like it was a convenient and understandable way to abbreviate percentile when squeezing it in the column title.
So you can keep your understanding unrevised approx. 99% of the time!